src/status_im/contexts/shell/activity_center/notification/community_request/view.cljs

```diff
@@ -56,27 +53,33 @@
                                     :t/joined-community
                                     :t/community-request-accepted-body-text)
                                   (when open? {:community community-name}))]
-                     community-context-tag]}
-
-      :else nil)))
+                     community-context-tag]})))
 
 (defn view
-  [{:keys [notification set-swipeable-height customization-color] :as props}]
+  [{:keys [notification extra-fn]}]
   (let [{:keys [community-id membership-status read
-                timestamp]}           notification
-        community                     (rf/sub [:communities/community community-id])
-        {:keys [header-text context]} (get-header-text-and-context community
-                                                                   membership-status)]
-    [swipeable props
-     [gesture/touchable-without-feedback
-      {:on-press (fn []
-                   (rf/dispatch [:navigate-back])
-                   (rf/dispatch [:communities/navigate-to-community-overview community-id]))}
+                timestamp]}   notification
+        community-name        (rf/sub [:communities/name community-id])
```

Comment:
Use small subscriptions instead of subscribing to the whole community plus destructuring. This should help users who stay in the AC while busy communities are being updated in the app-db via signals.